packages/client/app/routes.ts

```diff
@@ -1,3 +1,6 @@
-import { index, type RouteConfig } from "@react-router/dev/routes";
+import { index, type RouteConfig, route } from "@react-router/dev/routes";
 
-export default [index("routes/home.tsx")] satisfies RouteConfig;
+export default [
+	index("routes/home.tsx"),
+	route("/postQuestion", "routes/postQuestion.tsx"),
```

Comment:
```suggestion
	route("/posts/questions", "routes/post-question.tsx"),
```
REST APIではパスはリソースを指すから/postsっていリソース内の/questionsみたいな感じで書くと自然だと思う。あとファイル名はキャメルケースじゃなくてハイフンでつないでほしい！